setup.py

```diff
@@ -11,7 +11,7 @@
 
 setuptools.setup(
     name="testops-commons",
-    version="1.0.1",
+    version="1.0.5",
```

Comment:
consider for 1.1.0 because the third number is for the bug fixing version, second for minor feature, first for the major feature

Reply:
removed. Only update version when merge release

Reply:
Will consider the version number later